app/models.py

```diff
@@ -103,12 +104,16 @@ def query_in_deployment(cls):
         return cls.query.filter(cls.deployment.in_(current_app.config['SEARCH_DEPLOYMENTS']))
 
     @property
+    def full_name(self):
+        return u"%s %s" % (self.first_name, self.last_name)
+
+    @property
     def display_in_search(self):
         '''
         Determine whether user has filled out bare minimum to display in search
         results.
         '''
-        return self.first_name is not None and self.last_name is not None
+        return bool(self.first_name and self.last_name)
```

Comment:
This method is funky because anyone who has submitted the profile page form and left the first/last name blank now has a *blank* first and last name, which is not the same as a NULL first/last name... I've rewritten this method to hopefully be more accurate but let me know if my assumptions are totally off.